app/models/content_item.rb

```diff
@@ -10,10 +10,20 @@ def initialize(data)
   end
 
   def self.find!(content_id)
-    new Services.publishing_api.get_content(content_id).to_h
+    content_item = Services.publishing_api.get_content(content_id)
+    raise ItemNotFound unless content_item
+    new(content_item.to_h)
   end
 
   def link_set
     @link_set ||= LinkSet.find(content_id)
   end
+
+  def app_responsible_for_tagging
+    @tagging_apps ||= YAML.load_file("#{Rails.root}/config/tagging-apps.yml")
+    @tagging_apps[publishing_app]
+  end
+
+  class ItemNotFound < Exception
```

Comment:
Let's inherit from `StandardError` here, as it's an application-level exception rather than a system-level one.

Also, `ItemNotFoundError` would be a (slightly) more idiomatic name. And a bit of a nitpick, but placing it at the top of the class is a bit nicer as it's useful for any constant definitions to be quite obvious to the reader.